server/views/pages/bookAVisit/dateAndTime.njk

```diff
@@ -22,7 +22,7 @@
       {% if (doubleBooked and not checked) -%}
         Prisoner has a visit
       {%- elseif slot.availableTables <= 0 -%}
-        Fully booked
+        Fully booked ({{ slot.capacity + (slot.availableTables | abs) }} of {{ slot.capacity}} tables booked)
```

Comment:
If i was being really picky, I'd say you needed some code around the `s` in `tables` for when it's a single table available, but that can be done whenever as it's not going to happen very often, if ever 😁

Reply:
Fixed!